go/cmd/server/server_test.go

```diff
@@ -33,20 +42,40 @@ func getRepoPath(basePath string) string {
 }
 
 // Starts a new grpc server, registers the serving service and returns a client.
-func getClient(ctx context.Context, basePath string) (serving.ServingServiceClient, func()) {
+func getClient(ctx context.Context, basePath string, enableLogging bool) (serving.ServingServiceClient, func()) {
 	buffer := 1024 * 1024
 	listener := bufconn.Listen(buffer)
 
 	server := grpc.NewServer()
 	config, err := registry.NewRepoConfigFromFile(getRepoPath(basePath))
+
+	// TODO(kevjumba): either add this officially or talk in design review about what the correct solution for what do with path.
+	// Currently in python we use the path in FileSource but it is not specified in configuration unless it is using file_url?
+	if enableLogging {
+		if config.OfflineStore == nil {
+			config.OfflineStore = map[string]interface{}{
+				"path": ".",
+			}
+		} else {
+			config.OfflineStore["path"] = "."
+		}
+	}
+
 	if err != nil {
 		panic(err)
 	}
 	fs, err := feast.NewFeatureStore(config, nil)
 	if err != nil {
 		panic(err)
 	}
-	serving.RegisterServingServiceServer(server, &servingServiceServer{fs: fs})
+	generateFeatureService(fs)
```

Comment:
not sure what this function does if it returns nothing

Reply:
made function more descriptive.